kerastuner/engine/base_tuner.py

```diff
@@ -261,4 +261,4 @@ def get_trial_dir(self, trial_id):
     def _get_tuner_fname(self):
         return os.path.join(
             self.project_dir,
-            'tuner_' + str(self.tuner_id) + '.json')
+            str(self.tuner_id) + '.json')
```

Comment:
Why not the `tuner` prefix? It makes it easier for someone browsing the files to see what it is about.

Reply:
I was running this tuner with a few examples and realized that since our default `self.tuner_id` is `tuner0`, we were saving to the file `tuner_tuner0.json`. An alternative solution is we switch it so that the default tuner_id is just "0", WDYT?